other_projects_that_use_sympy.tex

```diff
@@ -0,0 +1,70 @@
+\subsection{SymPy Gamma}\label{sympy-gamma}
+
+SymPy Gamma is a simple web application that runs on Google App Engine 
+that executes and displays the results of SymPy expressions as well as
```

Comment:
runs on Google App Engine that executes --> runs on Google App Engine. It executes